sos/plugins/origin.py

```diff
@@ -119,6 +119,9 @@ def setup(self):
             # Note: Information about nodes, events, pods, and services
             # is already collected by the Kubernetes plugin
             self.add_cmd_output([
+                "openshift vesion",
+                "%s vesion" % self.oc_cmd,
+                "%s vesion" % self.oadm_cmd,
```

Comment:
Note that these will return basically the same information.
Moreover, we have version information from the rpm and/or docker plugins, so this sounds redundant, no?